packages/inquirer-common/src/prompts/utility.ts

```diff
@@ -75,5 +92,9 @@ export function ui5VersionsGrouped(
         );
     }
 
-    return [...maintChoices, ...notMaintChoices];
+    const versionChoices = [...maintChoices, ...notMaintChoices];
+    if (defaultChoice) {
+        versionChoices.unshift(defaultChoice);
+    }
```

Comment:
Coverage could be improved for `defaultChoice`

Reply:
Done thx

Reply:
Resolving with agreement